catt/cli.py

```diff
@@ -99,34 +95,16 @@ def cast(settings, video_url):
         click.echo("Serving local file, press Ctrl+C when done.")
         while thr.is_alive():
             time.sleep(1)
-    # Google blocks Chromecast Audio devices from running the YouTube app.
-    elif stream.is_youtube_video and cc_type != "audio":
-        click.echo("Casting YouTube video %s..." % stream.video_id)
+
+    elif stream.is_video and cst.name != "default":
```

Comment:
Why are we checking for `cst.name != "default"` here? This repeated check looks like it'd be better split off, perhaps?

Reply:
Also there should probably be a comment explaining this, as I'm sure others will have the same question.

Reply:
Yes, this is not done. I haven't yet made up my mind on how to solve this.

Reply:
Maybe move much of this code to a `play_stream` method that will be more explicit about its actions?